charts/bungee/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: bungee
+version: 1
```

Comment:
Apparently this needs to be a string

https://github.com/itzg/minecraft-server-charts/pull/50/checks?check_run_id=1842195069#step:4:104